src/public/map.js

```diff
@@ -78,9 +78,45 @@ function addMapEvents() {
   });
 }
 
+// function populateMenus(response) {
```

Comment:
should these commented-out functions be deleted, or are they works-in-progress that will be used soon?